cmds/core/fbsplash/main.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
main_linux.go. This will not work on most kernels.

Reply:
similar to the other code, I'm not sure this belongs in core.

Reply:
Yeah I'm not yet familiar with the structure. I had a brief chat with @hugelgupf. Regarding the kernel: This should also work on FreeBSD, where I have `/dev/fb0`. I will see how I can modularize this whole thing so it works for everyone. :)